datapower-docker-compose-dp-mq/datapower-docker-compose-dp-mq.mkd

```diff
@@ -0,0 +1,151 @@
+## A DataPower and MQ environment with Docker Compose 
+This sample demonstrates how to utilize multiple containers, orchestrated by Docker Compose, to create an environment that includes DataPower receiving and processing a request and writing the resulting message to an MQ back end. Subsequent support containers are included to send requests and store style sheets and schema utilized by the DataPower processing policy. In this sample, I'll outline the travels of the request message, `bob`, through the pipeline created in the docker-compose environment.
+
+This is a highly customizable example and it is encouraged that you play around with it.
+
+## Prerequisites
+Before you attempt this tutorial, please be sure that you:
+
+- Read, and better yet attempt, the [Docker Compose Hello World Application](https://github.com/ibm-datapower/datapower-tutorials/blob/master/datapower-docker-compose-hello-world/datapower-docker-compose-hello-world.md) sample. This is a great starting point that offers some of the detailed framework of how DataPower containers can be created and used in a continuous integration system.
+- For simplicity, in this sample, I created and used a Docker MQ base image. If you wish to run this step manually, please follow the instrucitons on [ibmcom/mq](https://hub.docker.com/r/ibmcom/mq/) on Docker Hub to create your own base Docker MQ image.
+
+
+## The Composed Application
+No rock band here, just the message Bob. Harley's Hello World example (mentioned in the Prerequisite section) goes into great detail about the various elements of his project. In order to avoid redundancy, I will not go into such detail here, but I will still cover the whole project. Let's take a look at the `docker-compose.yml` file that orchestrates the entire environment.
+```
+version: '2'
+services:
+    mq:
+      build: mq
+    http:
+      build: http
+    datapower:
+      build: datapower
+      depends_on:
+       - mq
+       - http
+    soapui:
+      image: ddavison/soapui:latest
+      depends_on:
+       - datapower
+    curldriver:
+      build: curldriver
+      depends_on:
+       - soapui
+```
+
+In all, the environment consists of five services, which build five images, and run five containers. 
+
+- The `curldriver` service builds an image with curl installed from an Ubuntu image.
```

Comment:
Tell us what it does as well. Perhaps `The curldriver service connects to soapui which drives the transaction. All curldriver does is send simple curl requests`